### DISCUSSION TITLE
Idea for the change character menu

### DISCUSSION THREAD

**Discussion by DeceptedPar** [MrMeowzz/Funkin-MrMeowzz#13] [Ideas]
maybe add in all of the story mode characters for a playable slot for their ogg songs like week 7 songs have so you can play the enemy's chart

**Comment by MrMeowzz** [comment]
What do you exactly mean?

**Comment by DeceptedPar** [comment]
what I mean is how you did it with tank man on week 7, you go to a tankman song, you choose tankman and you swap with tankman and you get his chart, I'm sorry if you don't understand this, I aint the best at explaining
